coalib/processes/BearRunning.py

```diff
@@ -98,7 +98,7 @@ def run_bear(message_queue, timeout, bear_instance, *args, **kwargs):
     if kwargs.get("dependency_results", True) is None:
         del kwargs["dependency_results"]
 
-    name = bear_instance.__class__.__name__
+    name = bear_instance.name
```

Comment:
it is important to note that we sometimes do `bear.__name__` those occasions have to stay like they are since they use the *class* not the *instance*